runtime/klee-libc/strtoul.c

```diff
@@ -57,7 +56,7 @@ strtoul(const char * nptr, char ** endptr, int base)
 	s = nptr;
 	do {
 		c = *s++;
-	} while (isspace((unsigned char)c));
+	} while (c == ' ');
```

Comment:
FreeBSD man for `isspace` says:

>      The isspace() and isspace_l() functions test for white-space characters.
>     For any locale, this includes the following standard characters:
>
>     ``\t''   ``\n''    ``\v''    ``\f''    ``\r''    `` ''

I think we should do the same here.

Reply:
Done!